tfswitch-completion.bash

```diff
@@ -0,0 +1,38 @@
+# Remove once https://github.com/warrensbox/terraform-switcher/issues/537 is implemented
+# shellcheck disable=SC2207 # W: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
+_tfswitch() {
+	local cur prev
+	cur=${COMP_WORDS[COMP_CWORD]}
+	prev=${COMP_WORDS[COMP_CWORD - 1]}
+
+	if [[ ${cur} == -* ]]; then
+		COMPREPLY=($(compgen -W "$(tfswitch --help 2>&1 | grep -Eo '[[:space:]]+(-{1,2}[a-zA-Z0-9-]+)')" -- "$cur"))
+		return 0
+	fi
+
+	case "${prev}" in
+	-A | --arch)
+		COMPREPLY=($(compgen -W "386 amd64 arm arm64" -- "$cur"))
+		return 0
+		;;
+	-b | --bin)
+		_filedir
```

Comment:
I think this is deprecated? https://github.com/scop/bash-completion/blob/721fa87c9c4ed5f2be8e890948f351f07731dcbb/bash_completion.d/000_bash_completion_compat.bash#L294

Reply:
Ah, great catch. Thanks. Though… it's complicated =)
Ubuntu 22.04 has v2.11 of scop/bash-completion and latest Homebrew have `bash-completion` v1.3 because of macOS's version of Bash (and folks have to install `bash-completion@2` if they have Bash upgraded).
So I implemented 524d1cd to support backward compatibility. WDYT?